digital_land/update.py

```diff
@@ -1,44 +1,42 @@
 from datetime import datetime, date
-from digital_land.register import Item
-from digital_land.collection import LogRegister, EndpointRegister, SourceRegister
-from pathlib import Path
-import hashlib
+
+from .register import Item, hash_value
+from .collection import Collection
+from .schema import Schema
 
 
 def get_source_endpoint_fieldnames():
```

Comment:
I'm not sure it's worth having this as a function here. It's only used once by cli (and unlikely to be used a lot more I expect). We could instead just do the `set(...)` directly in cli.py which would make that code slightly more readable (as it would be local in the function) and an added bonus that we can delete the dependency on Schema from update.py